fixtures/ios_app/xx.ipa/DistributionSummary.plist

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we use a name other than `xx.ipa?

Reply:
oops, just some leftover from debugging I missed, let me remove it